apps/breethe/lib/breethe/sources/eea.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Breethe.Sources.EEA do
+  alias __MODULE__.CSV
+
+  def country_codes(),
+    do: [
+      "BG",
+      "CY",
+      "CZ",
+      "ES",
+      "GR",
+      "HR",
+      "HU",
+      "IE",
+      "IS",
+      "IT",
+      "LT",
+      "LU",
+      "NO",
+      "PT",
+      "RS",
+      "SE",
+      "SK",
+      "AD",
+      "BA",
+      "BE",
+      "DE",
+      "FI",
+      "FR",
+      "GB",
+      "GI",
+      "LV",
+      "MT",
+      "NL",
+      "SI",
+      "DK",
+      "GE",
+      "MK",
+      "PL",
+      "AT",
+      "CH",
+      "EE"
+    ]
+
+  # @parameters ["PM10", "PM2.5", "SO2", "NO2", "O3", "CO"]
+  def parameters(), do: ["SO2", "NO2", "O3", "CO"]
```

Comment:
Why no `@parameters`?

Reply:
I exposed the `params` and the `country_codes` in functions to avoid code duplication in the tests, not sure if this is a good approach though?

Reply:
Yeah, makes sense - I think a more common idiom is

```elixir
@parameters ["PM10", "PM2.5", "SO2", "NO2", "O3", "CO"]
def parameters(), do: @parameters
```

though?

Reply:
fixed